File: yht0827/1월/1월 31일/바탕화면 정리.java
class Solution {
	public int[] solution(String[] wallpaper) {
		int minRow, minCol;
		int maxRow, maxCol;

		minRow = minCol = Integer.MAX_VALUE;
		maxRow = maxCol = Integer.MIN_VALUE;

		int row = wallpaper.length;
		int col = wallpaper[0].length();

		for (int r = 0; r < row; r++) {
			for (int c = 0; c < col; c++) {
				if (wallpaper[r].charAt(c) == '#') {
					minRow = Math.min(minRow, r);
					minCol = Math.min(minCol, c);
					maxRow = Math.max(maxRow, r);
					maxCol = Math.max(maxCol, c);
				}
			}
		}

		return new int[] {minRow, minCol, maxRow + 1, maxCol + 1};
	}
}

File: zuzubibi/old/1월/1월 31일/바탕화면 정리.py
def solution(wallpaper):
    x, y = [], []
    for i in range(len(wallpaper)):
        for j in range(len(wallpaper[0])):
            if wallpaper[i][j] =='#':
                x.append(i)
                y.append(j)

    return [min(x), min(y), max(x)+1, max(y)+1]


print(solution([".#...", "..#..", "...#."])) # [0, 1, 3, 4]
print(solution(["..........", ".....#....", "......##..", "...##.....", "....#....."])) # [1, 3, 5, 8]
print(solution([".##...##.", "#..#.#..#", "#...#...#", ".#.....#.", "..#...#..", "...#.#...", "....#...."])) # [0, 0, 7, 9]
print(solution(["..", "#."])) # [1, 0, 2, 1]
